docs/user/dql/functions.rst

```diff
@@ -1797,6 +1797,52 @@ Example::
     +------------------+----------------+
 
 
+PERIOD_ADD
+----------
+
+Description
+>>>>>>>>>>>
+
+Usage: period_add(P, N) add N months to period P (in the format YYMM or YYYYMM). Returns a value in the format YYYYMM.
+
+Argument type: LONG, LONG
+
+Return type: LONG
```

Comment:
Any thought on whether we should make it LONG or just INTEGER? Because the possible number seems very small.

Reply:
Good point. It could be even SHORT.
I was misled by `mysql`'s output (`mysql --column-type-info`):
```
mysql> select period_add(207704, 15), period_diff(202210, 202003);
Field   1:  `period_add(207704, 15)`
Catalog:    `def`
Database:   ``
Table:      ``
Org_table:  ``
Type:       LONGLONG
Collation:  binary (63)
Length:     21
Max_length: 6
Decimals:   0
Flags:      NOT_NULL BINARY NUM

Field   2:  `period_diff(202210, 202003)`
Catalog:    `def`
Database:   ``
Table:      ``
Org_table:  ``
Type:       LONGLONG
Collation:  binary (63)
Length:     21
Max_length: 2
Decimals:   0
Flags:      NOT_NULL BINARY NUM


+------------------------+-----------------------------+
| period_add(207704, 15) | period_diff(202210, 202003) |
+------------------------+-----------------------------+
|                 207807 |                          31 |
+------------------------+-----------------------------+
1 row in set (0.00 sec)
```
I used LONG, since it was reported here. Actually, MySQL [has no LONG](https://dev.mysql.com/doc/refman/8.0/en/integer-types.html).
I'm ready to change signatures to `(SHORT, SHORT) -> SHORT` or `(INT, INT) -> INT` if needed. On another hand, I don't see any profit.

Reply:
@dai-chen,
Should I change signature or we can keep this as is?

Reply:
yeah, if so, IMO SHORT or INT is not only space saving but also more clear to user.

Reply:
`SHORT` is too short, unfortunately. Changed to `INT` in 957c00fc.

Reply:
Sounds good. Thanks!